util/process_wrapper/BUILD.tinyjson

```diff
@@ -0,0 +1,7 @@
+load("@//rust/private:rust.bzl", "rust_library_without_process_wrapper")
```

Comment:
I know this PR is WIP but just so I don't forget, can you rename this file to `BUILD.tinyjson.bazel` so it's picked up by buildifier?